src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -941,11 +941,7 @@ protected override void EndProcessing()
                 PathUtils.MasterStreamOpen(
                     this,
                     filePath,
-#if UNIX
                     new UTF8Encoding(false), // UTF-8, no BOM
```

Comment:
Would prefer using named parameter instead of a comment about no BOM.

Reply:
Will update